doc/input-output-reference/src/overview/group-coil-cooling-dx.tex

```diff
@@ -64,6 +64,33 @@ \subsubsection{Inputs}\label{inputs-01}
 
 This field is optional. It is used to describe where the coil obtains water used for evaporative cooling of its condenser. If blank or omitted, then the unit will obtain water directly from the mains. If the name of a WaterUse:Storage object is used here, then the unit will obtain its water from that tank. If a tank is specified, the unit will attempt to obtain all the water it uses from the tank. However, if the tank cannot provide all the water the condenser needs, then the unit will still operate and obtain the rest of the water it needs from the mains (referred to as StarvedWater).
 
+\subsection{Coil:Cooling:DX:ASHRAE205:Performance}\label{coilcoolingdxashrae205performance}
+
+This object defines basic unit control and ASHRAE205 representation file for one Coil:Cooling:DX object. 
+
+\subsubsection{Inputs}
+
+\paragraph{Field: Name}
+
+A unique user-assigned name for an instance of a cooling coil performance specification. Any reference to this cooling coil performance specification by a cooling coil object will use this name.
+
+\paragraph{Field: Representation File Name}
+
+The filename of the ASHRAE205 chiller representation (.json or .cbor).
+
+\paragraph{Field: Performance Interpolation Method}
+
+This alpha field specifies how performance data from the Representation File will be interpolated.
+Choices are "Linear" and "Cubic."
```

Comment:
For the record, in LaTeX, we prefer ``backticks then apostrophes'' and let TeX render them as it sees fit.  Don't need to hold this up for it.